sdk/communication/Azure.Communication.AlphaIds/tests.yml

```
@@ -0,0 +1,18 @@
+trigger: none
+
+extends:
+  template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+  parameters:
+    ServiceDirectory: communication
+    Project: Azure.Communication.ShortCodes
```

I believe this is the wrong project? Should be AlphaIds.

```suggestion
    Project: Azure.Communication.AlphaIds
```

---

Good catch, my mistake. Just fixed it.